types.go

```diff
@@ -325,15 +325,20 @@ type AlertChannelWebhook struct {
 // AlertChannel represents an alert channel and its subscribed checks. The API
 // defines this data as read-only.
 type AlertChannel struct {
-	ID        int64                 `json:"id,omitempty"`
-	Type      string                `json:"type"`
-	CreatedAt time.Time             `json:"created_at"`
-	UpdatedAt time.Time             `json:"updated_at"`
-	Email     *AlertChannelEmail    `json:"-"`
-	Slack     *AlertChannelSlack    `json:"-"`
-	SMS       *AlertChannelSMS      `json:"-"`
-	Opsgenie  *AlertChannelOpsgenie `json:"-"`
-	Webhook   *AlertChannelWebhook  `json:"-"`
+	ID                 int64                 `json:"id,omitempty"`
+	Type               string                `json:"type"`
+	CreatedAt          time.Time             `json:"created_at"`
+	UpdatedAt          time.Time             `json:"updated_at"`
+	Email              *AlertChannelEmail    `json:"-"`
+	Slack              *AlertChannelSlack    `json:"-"`
+	SMS                *AlertChannelSMS      `json:"-"`
+	Opsgenie           *AlertChannelOpsgenie `json:"-"`
+	Webhook            *AlertChannelWebhook  `json:"-"`
+	SendRecovery       bool                  `json:"sendRecovery"`
```

Comment:
Khaled we should make all these fields pointers. It then gives more power to the backend to handle the cases when they are not set

Reply:
done